tests/versioned/mysql2/package.json

```diff
@@ -20,10 +20,10 @@
     },
     {
       "engines": {
-        "node": ">=4.0"
+        "node": ">=12.0"
```

Comment:
the minor version split here looks a bit weird. did the mysql2 module drop support for older versions of node in a minor release?

Reply:
https://github.com/sidorares/node-mysql2/blob/master/Changelog.md

1.7.0
  - Fix crashing when session info packet does not 
    start with length-coded string                    #1004, #989
  - build: drop node 4 and 6 and add node v12         #997
  - Add support for timezone connection option        #996, #15, #262, 
                                                      #642, #877, #888
  - Make mysql2 compatible with minification          #992, #890, #899, 
                                                      #890
  - fix serialisation of '00:00:00' time              #968, #967
  - Allow to set minVersion ssl option                #961, #960
  - Fix a MaxListenersExceededWarning with stream 
    local infile                                      #965

Reply:
I guess that may be "build" only. Although, that also seems the only points by which you can be confident those work?